source/includes/changelg.md

```diff
@@ -0,0 +1,15 @@
+# Changelog
```

Comment:
I don't think we have the data to back this up. I know we had this initially, but it was removed.